packages/js/product-editor/src/hooks/use-error-handler.ts

```diff
@@ -65,7 +65,9 @@ function getErrorPropsWithActions(
 					navigateTo( {
 						url: getUrl( errorContext ),
 					} );
-					focusByValidatorId( validatorId );
+					setTimeout( () => {
```

Comment:
Why was this added? Was it not working with the `InputControl` without the setTimeout?

Reply:
Exactly, we need to focus on the component a few milliseconds later in order to have the component printed on the screen. Otherwise, it won't work.